lib/gui/app/components/file-selector/controllers/file-selector.js

```diff
@@ -50,9 +50,10 @@ module.exports = function (
    */
   this.getFolderConstraints = utils.memoize(() => {
```

Comment:
This used to return an array, but now returns a string, so perhaps it should be renamed `getFolderConstraint` ?
And should the `TODO` line now be removed?

Reply:
Yup, done!